### DISCUSSION TITLE
AI人工智慧導論期末專題參考資源

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS3073-C/Python-Community-GS3073-C#238] [General]
今天(2022/2/15(二)老師提到期末專題的參考資源，有上學期班級的youtube影片。因為影片的權限是公開(public)的，因此直接查詢ncuedu便可直接查到。在youtube查詢ncuedu的結果如下
![image](https://user-images.githubusercontent.com/86505454/154042642-7f6cece6-71e3-437b-86e4-3bc2f8f16ed9.png)
我也直接在下面附上此ncuedu的頻道(channel)網站
[https://www.youtube.com/channel/UCQJB5pz69XaB8f7ewJddd7Q/featured](https://www.youtube.com/channel/UCQJB5pz69XaB8f7ewJddd7Q/featured)

**Comment by chia-kai-chang** [comment]
謝謝 @enoch-lightning 的補充
